### ISSUE TITLE
unspecific EXIF message during import of NEF or NRW files

### ISSUE THREAD

**Issue by Macchiato17** [darktable-org/darktable#17731]
### Describe the bug

I’m using darktable 4.9 current master. For some months now, I’m getting a puzzling message when importing my Nikon NEF or NRW files into lighttable. For each imported picture I get one message

```EXIF tag not found in```

I can’t see any issue with the EXIF tags of the imported RAWs, and as I haven’t changed neither body nor lens nor camera configuration, I simply don’t understand the meaning of this message. I tried to do some logging, but I don’t get anything from the logs that would point to a related issue.

### Steps to reproduce

Please use the NEFs which I provided in #17684: just import them and see the above mentioned message.

### Expected behavior

provide a more specific message, if anything is wrong with the EXIF content

### Logfile | Screenshot | Screencast

[darktable-log.txt](https://github.com/user-attachments/files/17534041/darktable-log.txt)

### Where did you obtain darktable from?

self compiled

### darktable version

4.9.0+925~gb7288e18b2

### What OS are you using?

Windows

### What is the version of your OS?

Windows 11

### Describe your system?

Laptop, Intel i5 8265U, 8GB RAM, nvidia GTX 1050 Max-Q Design (4GB)

### Are you using OpenCL GPU in darktable?

Yes

### If yes, what is the GPU card and driver?

nvidia GTX 1050 Max-Q Design (4GB), driver 561.09

**Comment by pehar1** [comment]
4.9.0+943~gb8b70e4786, Ubuntu 24.04
Imported your image D75_2876.NEF with and without related sidecar (add to library). Import successful without EXIF related message  you report. This is the output of -d common importing the image without xmp (for the case you want to look yourself or compare with your import) :
[log-d75_2876.txt](https://github.com/user-attachments/files/17538547/log-d75_2876.txt)

**Comment by pehar1** [comment]
As far as I can tell, the message "`EXIF tag not found in`" comes from the LUA script `autostyle.lua` (https://github.com/darktable-org/lua-scripts/blob/3a1f6bbc7a19e9c61015e8bf0c22c6668ba4cbc0/contrib/autostyle.lua) and it seems to be related to the application of a style (I hope I correctly understand the code...). @wpferguson , may I ask for your expertise ?

Against which exiv2 version (`libexiv2-dev` on Linux, not sure about the name of the library on Windows) did you build darktable ? I build against 0.27.6.

**Comment by Macchiato17** [comment]
OMG guys, you are so lightning fast - sorry for not replying earlier - perhaps due to timezones 😁 

@pehar1 I'm building against exiv2 0.28.3-1 (but as I'm writing this, it's already no longer important 🤪)
@wpferguson I was not aware that the LUA script AutoStyle was indeed active on my installation. Don't know that / when I activated it. But yes, I just found that the button is "on". So I switched it off. But as this issue has already been merged and closed, you were faster than me 😎

Thanks to all of you fantastic guys for supporting me so quickly with that issue.
